app/src/main/java/com/cornellappdev/uplift/ui/screens/LandingScreen.kt

```diff
@@ -0,0 +1,72 @@
+package com.cornellappdev.uplift.ui.screens
+
+import androidx.compose.animation.Crossfade
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.platform.LocalContext
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavHostController
+import com.cornellappdev.uplift.models.LocationRepository
+import com.cornellappdev.uplift.networking.ApiResponse
+import com.cornellappdev.uplift.ui.screens.subscreens.MainError
+import com.cornellappdev.uplift.ui.screens.subscreens.MainLoaded
+import com.cornellappdev.uplift.ui.screens.subscreens.MainLoading
+import com.cornellappdev.uplift.ui.viewmodels.ClassDetailViewModel
+import com.cornellappdev.uplift.ui.viewmodels.GymDetailViewModel
+import com.cornellappdev.uplift.ui.viewmodels.HomeViewModel
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+import com.google.accompanist.permissions.rememberMultiplePermissionsState
+import com.valentinilk.shimmer.Shimmer
+import kotlinx.coroutines.delay
+import android.util.Log
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.uplift.R
+
+/**
+ * The landing page during the Uplift onboarding process.
+ */
+@OptIn(ExperimentalPermissionsApi::class)
+@Composable
+fun LandingScreen() {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(bottom = 54.dp),
+        verticalArrangement = Arrangement.Bottom,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Image(
+            painter = painterResource(id = R.drawable.ic_main_logo),
+            contentDescription = "Uplift logo",
+            modifier = Modifier
+                .height(130.dp)
+                .width(115.dp)
+        )
+
+        Spacer(modifier = Modifier.height(387.dp));
+
+        Image(
+            painter = painterResource(id = R.drawable.ic_appdev_logo),
+            contentDescription = "Cornell AppDev logo",
+            modifier = Modifier
+                .height(26.dp)
+                .width(158.dp)
+        )
```

Comment:
To make the bottom part equidistant from the bottom on all screen sizes, convert the outer container to a box, and then make the uplift logo padded relative to the top, and the cornell appdev logo padded relative to the bottom.

Reply:
done